src/test/java/com/thealgorithms/ciphers/XORCipherTest.java

```diff
@@ -0,0 +1,19 @@
+package com.thealgorithms.ciphers;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+class XORCipherTest {
+
+    @Test
+    void shouldEncryptAndDecryptTest() {
```

Comment:
Please also add tests showing that the encrypted text is different from the source text